test/Microsoft.Sbom.Api.Tests/Utils/ComponentDetectionCliArgumentBuilderTests.cs

```diff
@@ -146,27 +146,21 @@ public void Build_AddDetectorArgsViaAddArgCombineWithOtherDetectorArgs()
     }
 
     [TestMethod]
-    [ExpectedException(typeof(ArgumentNullException))]
     public void Build_WithNullValue()
     {
-        var builder = new ComponentDetectionCliArgumentBuilder()
+        Assert.ThrowsException<ArgumentNullException>(() => new ComponentDetectionCliArgumentBuilder()
             .SourceDirectory("X:/")
             .AddArg("ManifestFile", null)
-            .AddArg("--DirectoryExclusionList", "X:/hello");
-
-        builder.Build();
+            .AddArg("--DirectoryExclusionList", "X:/hello"));
```

Comment:
For both these tests, original code makes it look like the method under test was Build(), but it is no longer called in the new code. Is that expected?

Reply:
The exception is occurring in the constructor. It doesn't feel like that was the original intent so perhaps the original test was not actually testing what it was supposed to test

Reply:
The Exceptions were actually coming from the `AddArg` method. I've renamed these tests to reflect what is actually being called and scoped them down to just the code that is needed